moderation-commands/infractions.md

```diff
@@ -48,14 +48,14 @@ MoonlightBot requires the following permissions to successfully execute this sub
 
 ## reason
 
-<TODO: describe>
+Change the reason of the infraction. If the `reason` option is left blank, it will simply display the current one
```

Comment:
```suggestion
Changes the reason of the infraction. If the `reason` option is left blank, it will simply display the current one
```

also specify the behavior of putting the `%%%` placeholder inside the reason (refer to reminder edit, works the same way)